rt-fs/commands/cat.go

```diff
@@ -0,0 +1,121 @@
+package commands
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/jfrog/jfrog-cli-core/artifactory/commands"
+	"github.com/jfrog/jfrog-cli-core/artifactory/commands/generic"
+	"github.com/jfrog/jfrog-cli-core/artifactory/spec"
+	"github.com/jfrog/jfrog-cli-core/artifactory/utils"
+	clientutils "github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+
+	"github.com/jfrog/jfrog-cli-core/plugins/components"
+)
+
+const (
+	downloadThreads      = 3
```

Comment:
Since we're always downloading only 1 file, maybe we should change the value to 1.